### ISSUE TITLE
Sticky trap project management

### ISSUE THREAD

**Issue by rvosa** [naturalis/imgpheno#9]
- [ ] how many traps total?
- [ ] size difference?

**Comment by rvosa** [comment]
e6f7249499d34fb356b587f79c3061e867060fb8

**Comment by rl-michels** [comment]
I have received an answer from Jinze:
The idea is that these traps will be deployed throughout the netherlands, so a very large amount of traps will have to be analysed in the future. 
There are also a lot of different shapes and sizes of sticky traps available on the market, and there has not yet been a decision on which traps to use in the large scale deployment. Boki and Jinze might look into deciding which trap to use in a short time, so i will have time to implement those dimensions in the program.

in short:
-there will be a lot of traps
-as of yet there might be the possibillity of dealing with different sizes.

**Comment by rvosa** [comment]
For the project plan, it would be great if it included reasonable suggestions for how and where the code that gets written can be incorporated into the existing imgpheno packages. The idea is that the functionality is for the most part generic enough so that it ought to be possible to incorporate most of it without sticking out like a sore thumb. Then there might still be some specific scripts to handle sticky trap data, but these would mostly be wrappers around reusable imgpheno code.

**Comment by rvosa** [comment]
Last steps for the sticky traps wrap up. Concerning data:
- [ ] remove `photos`,  `photos_original` and hidden .R folders from  sticky-trap-data
- [ ] add "real" photos from the instance to stick-trap-data
- [ ] add photo file name column to TSV data files
- [ ] add remaining data (photos, manual counts, project output) to nbclassify-data/archive-stickytraps

Concerning the instance:
- [ ] any code, data, artifacts on there needs to be committed in the appropriate repos
- [ ] package the virtual machine to a single file and tell me the size
